iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensatorAdder.java

```diff
@@ -36,6 +36,10 @@ default ShuntCompensatorAdder setRegulatingTerminal(Terminal regulatingTerminal)
         throw new UnsupportedOperationException();
     }
 
+    default ShuntCompensatorAdder useLocalRegulation(boolean use) {
+        throw new UnsupportedOperationException();
```

Comment:
Like in `GeneratorAdder`, put a default implementation that does nothing

Reply:
Done